zebra-chain/src/keys/sapling.rs

```diff
@@ -124,7 +124,9 @@ fn jubjub_group_hash(d: [u8; 8], m: &[u8]) -> Option<jubjub::ExtendedPoint> {
 ///
 /// [0]: https://github.com/zcash/librustzcash/blob/master/zcash_primitives/src/jubjub/mod.rs#L409
 /// https://zips.z.cash/protocol/protocol.pdf#concretegrouphashjubjub
-fn find_group_hash(d: [u8; 8], m: &[u8]) -> jubjub::ExtendedPoint {
+// TODO: move common functions like these out of the keys module into
+// a more appropriate location
```

Comment:
following on from this PR, what if we made top-level `sprout` and `sapling` modules, then had `zebra_chain::sapling::{keys, crypto::{ /* various stuff */ } }`?

Reply:
That's the goal yeah, after this gets merged we can re-org: https://github.com/ZcashFoundation/zebra/issues/567